src/platform/testing/e2e/cypress/support/form-tester/index.js

```diff
@@ -146,7 +146,17 @@ const processPage = () => {
   cy.location('pathname', COMMAND_OPTIONS).then(pathname => {
     if (pathname.endsWith('review-and-submit')) {
       performPageActions(pathname, false);
-      cy.findByLabelText(/accept/i).click();
+
+      // Check the privacy agreement box if it exists.
+      cy.get(APP_SELECTOR, COMMAND_OPTIONS).then($form => {
+        const privacyAgreement = $form.find('input[name^="privacyAgreement"]');
```

Comment:
At first glance I noticed this is an [anti-pattern](https://docs.cypress.io/guides/references/best-practices.html#Assigning-Return-Values) in Cypress, but I think it's okay since `privacyAgreement` doesn't have anything chained off of it.

Reply:
This particular case isn't because it's not assigning the result of a Cypress command. It's jQuery when it's inside the callback for a `then` command

Reply:
Ohh yeah, that's true!